posts/2023-03-31-embrace-kotlin.adoc

```diff
@@ -1,41 +1,44 @@
 = Embrace Kotlin: Where to Get Started for Scala Developers
 AnastasiiaL
 v1.0, 2023-03-31
-:title: Embrace Kotlin: Tips & Tricks for Scala Developers to Get Started
+:title: Embrace Kotlin: Tips And Tricks for Scala Developers to Get Started
 :imagesdir: ../media/2023-03-31-embrace-kotlin
 :lang: en
 :tags: [scala, kotlin]
 
 Recently I have been intrigued by the buzz surrounding Kotlin, another powerful language on the JVM.
-The growing popularity and claims of a seamless transition from Scala to Kotlin have piqued my curiosity,prompting me to dive in and explore the language myself.
-In this blog post, I'll share my journey of learning Kotlin from a Scala developer's perspective and provideinsights into the similarities and differences between the two languages that make the switch feel so smooth.
+The growing popularity and claims of a seamless transition from Scala to Kotlin have piqued my curiosity, prompting me to dive in and explore the language myself.
+In this blog post, I'll share my journey of learning Kotlin from a Scala developer's perspective and provide insights into the similarities and differences between the two languages that make the switch feel so smooth.
 
 == Language similarities
 
-Both Kotlin and Scala are statically-typed languages that run on the Java Virtual Machine. They share many features, such as:
-- Support for both functional programming and OO
-- Interoperability with Java
-- Type inference
-- Null safety
-- Extension functions
-- Pattern matching (although more limited in Kotlin)
+Both Kotlin and Scala are statically-typed languages that run on the Java Virtual Machine. They share many features, such as: +
```

Comment:
```suggestion
Both Kotlin and Scala are statically-typed languages that run on the Java Virtual Machine. They share many features, such as:
```

Reply:
without the plus there's no new line. Check how it looks right now here https://blog.acceptance.lunatech.com/posts/2023-03-31-embrace-kotlin